client/js/uploader.basic.api.js

```diff
@@ -337,6 +343,20 @@
             this._totalProgress && this._totalProgress.reset();
         },
 
+        resumeQueue: function() {
+            this._isPausedQueue = false;
+
+            // If upload has been paused, resume from the id it has been paused
+            // This will also upload all the files in the _waiting queue
+            if (this._pausedId !== -1) {
+                this._uploadFile(this._pausedId);
+            }
+
+            if (this._storedIds.length) {
```

Comment:
I'm not sure I understand why this action is being taken here. `uploadStoredFiles` applies to an entirely different behavior, and is not really related to pausing an in-progress queue. The decision to upload "stored" files should be made by the integrator through the API.